### PR TITLE
fix(ChatMessage): add prop to proptypes

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Hot fix for handled prop

#### Focus areas to test

(optional)

### CODE REVIEW

**Review by pompomon — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

File: packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const DatepickerCellExample = () => {
       calendar={{
         calendarCell: {
           children: (ComponentType, props) =>
-            props.isToday ? (
```
Comment (pompomon) [inline packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx:11]: Why this change is there?
@rymeskar , please did you merge your last PR?
